plugins/GSdx/GSHwHack.cpp

```diff
@@ -1370,13 +1370,15 @@ bool GSC_GT4(const GSFrameInfo& fi, int& skip)
 	// Game requires to extract source from RT (block boundary) (texture cache limitation)
 	if(skip == 0)
 	{
-		if(fi.TME && fi.FBP >= 0x02f00 && fi.FPSM == PSM_PSMCT32 && (fi.TBP0 == 0x00000 || fi.TBP0 == 0x01180 /*|| fi.TBP0 == 0x01a40*/) && fi.TPSM == PSM_PSMT8) //TBP0 0x1a40 progressive
-		{
-			skip = 770;	//ntsc, progressive 1540
-		}
-		if(g_crc_region == CRC::EU && fi.TME && fi.FBP >= 0x03400 && fi.FPSM == PSM_PSMCT32 && (fi.TBP0 == 0x00000 || fi.TBP0 == 0x01400 ) && fi.TPSM == PSM_PSMT8)
-		{
-			skip = 880;	//pal
+		if(Aggressive) { // Can fix or cause VRAM and RAM spikes. Depends on userconfig , game version and maybe hardware.
```

Comment:
`// Removes layer obscuring the screen when the in-game brightness or contrast setting is set to any value but 0. The hack can cause VRAM/RAM spikes when both brightness and contrast settings are set to 0.`